src/v2i-hub/RSUHealthMonitorPlugin/manifest.json

```diff
@@ -19,7 +19,7 @@
         },
         {
             "key":"RSUConfigurationList",
```

Comment:
Can we add the list of all possible authProtocol and privacyProtocol in the description for users?

Reply:
Good call